server/src/utils/date_time.rs

```diff
@@ -0,0 +1,15 @@
+//! Utilities to deal with date-time types ( DateTime, Instant, SystemTime, etc)
+
+use std::time::{Instant, SystemTime};
+
+/// Converts an Instant into a SystemTime.
+pub fn instant_to_system_time(instant: &Instant) -> SystemTime {
```

Comment:
`Instant` implements `Copy`, so you can pass by value here. Clippy will most likely complain about this.

Reply:
ok, done